crates/db_schema/src/impls/post.rs

```diff
@@ -104,7 +104,14 @@ impl Post {
       .filter(post::local.eq(true))
       .filter(post::deleted.eq(false))
       .filter(post::removed.eq(false))
-      .filter(post::published.ge(Utc::now().naive_utc() - Duration::days(SITEMAP_DAYS)))
+      .filter(
+        post::published.ge(
+          Utc::now().naive_utc()
+            - TimeDelta::try_days(SITEMAP_DAYS)
+              .ok_or("TimeDelta out of bounds")
+              .map_err(|e| Error::QueryBuilderError(e.into()))?,
```

Comment:
I would directly declare `SITEMAP_DAYS` as `TimeDelta` instead of int, would make the code here more concise. Then use expect as the value is hardcoded anyway.